docker/Dockerfile

```diff
@@ -53,6 +53,14 @@ RUN set -e; \
 
 RUN set -e; \
     export DEB_MAX_ATTEMPT=10; \
+    export CRIS_IMG_ARCH=$(case $(uname -m) in \
```

Comment:
double quote

Reply:
On that, I remebered that `$()` is the same as `"$()"`, isn't it?

Reply:
No

Reply:
Try `$(echo 'rm -rf /')` (please don't)